pkg/fleet/installer/service/datadog_installer_windows.go

```diff
@@ -65,7 +77,13 @@ func StopInstallerExperiment(ctx context.Context) (err error) {
 		}
 		span.Finish(tracer.WithError(err))
 	}()
-	err = msiexec("stable", datadogInstaller, "/i", nil)
+	cmd, err := msiexec("stable", datadogInstaller, "/i", nil)
+	if err == nil {
+		// This command will kill this process
+		// launch it asynchronously and don't care about its result
+		// because it will be 0xc000013a
```

Comment:
Could we check for this error ( 0xc000013a) specifically and have special handling for it?